packages/twig-new/src/bridge/index.js

```diff
@@ -0,0 +1,91 @@
+import { schema } from "./schema";
```

Comment:
Why `src/bridge`? It's a utility function, but it's not getting exported. May be `lib/compose` or something?